draft-ietf-dnsop-rfc7958bis.mkd

```diff
@@ -419,6 +421,16 @@ This document aims to specify carefully the means by which such trust
 anchors are published, with the goal of making it easier for those
 trust anchors to be integrated into user environments.
 
+# IANA Considerations
+
+Each time IANA produces a new trust anchor,
+it SHOULD publish it using the format described in this document.
```

Comment:
"When a new trust anchor produced by IANA is ready for use, IANA MUST publish it using the format described in this document."

Reply:
Agree. A MUST is better than a SHOULD.